class-jetpack-currencies.php

```diff
@@ -0,0 +1,175 @@
+			'symbol'  => 'NZ$',
+			'decimal' => 2,
+		),
+		'AUD' => array(
+			'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+			'symbol'  => 'A$',
+			'decimal' => 2,
+		),
+		'CAD' => array(
+			'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+			'symbol'  => 'C$',
+			'decimal' => 2,
+		),
+		'ILS' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => '₪',
+			'decimal' => 2,
+		),
+		'RUB' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => '₽',
+			'decimal' => 2,
+		),
+		'MXN' => array(
+			'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+			'symbol'  => 'MX$',
+			'decimal' => 2,
+		),
+		'MYR' => array(
+			'format'  => '%2$s%1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'RM',
+			'decimal' => 2,
+		),
+		'SEK' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'Skr',
+			'decimal' => 2,
+		),
+		'HUF' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'Ft',
+			'decimal' => 0, // Decimals are supported by Stripe but not by PayPal.
+		),
+		'CHF' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'CHF',
+			'decimal' => 2,
+		),
+		'CZK' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'Kč',
+			'decimal' => 2,
+		),
+		'DKK' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'Dkr',
+			'decimal' => 2,
+		),
+		'HKD' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'HK$',
+			'decimal' => 2,
+		),
+		'NOK' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'Kr',
+			'decimal' => 2,
+		),
+		'PHP' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => '₱',
+			'decimal' => 2,
+		),
+		'PLN' => array(
+			'format'  => '%2$s %1$s', // 1: Symbol 2: currency value
+			'symbol'  => 'PLN',
+			'decimal' => 2,
+		),
+		'SGD' => array(
+			'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+			'symbol'  => 'S$',
+			'decimal' => 2,
+		),
+		'TWD' => array(
+			'format'  => '%1$s%2$s', // 1: Symbol 2: currency value
+			'symbol'  => 'NT$',
+			'decimal' => 0, // Decimals are supported by Stripe but not by PayPal.
+		),
+		'THB' => array(
+			'format'  => '%2$s%1$s', // 1: Symbol 2: currency value
+			'symbol'  => '฿',
+			'decimal' => 2,
+		),
+	);
+
+	/**
+	 * Format a price with currency.
+	 *
+	 * Uses currency-aware formatting to output a formatted price with a simple fallback.
+	 *
+	 * Largely inspired by WordPress.com's Store_Price::display_currency
+	 *
+	 * @param  string $price    Price.
+	 * @param  string $currency Currency.
+	 * @param  bool   $symbol   Whether to display the currency symbol.
+	 * @return string           Formatted price.
+	 */
+	public static function format_price( $price, $currency, $symbol = true ) {
```

Comment:
This ones comes from `Jetpack_Simple_Payments` too: https://github.com/Automattic/jetpack/pull/17381/files#diff-b73125af4e0fe93b3939f29ff9c7e6fc33d17e564a32fb7ea8d082d1009e3401